### PR TITLE
Chore/add state icons

### PR DESCRIPTION
**Pull request by trishtzy**

### CODE REVIEW

**Comment by trishtzy** [comment]
awaiting-drop-off.svg:
Done in 19 ms!
0.991 KiB - 50.8% = 0.487 KiB

awaiting-collection.svg:
Done in 24 ms!
1.343 KiB - 39.8% = 0.809 KiB

booked-for-delivery.svg:
Done in 28 ms!
1.178 KiB - 25.6% = 0.876 KiB

completed.svg:
Done in 20 ms!
0.434 KiB - 40.5% = 0.258 KiB

customs-awaiting-payment.svg:
Done in 27 ms!
2.162 KiB - 27.6% = 1.564 KiB

customs-on-hold.svg:
Done in 27 ms!
2.163 KiB - 27.7% = 1.564 KiB

delivery-attempted.svg:
Done in 47 ms!
4.615 KiB - 50.1% = 2.302 KiB

damaged.svg:
Done in 22 ms!
0.991 KiB - 50.8% = 0.487 KiB

in-transit.svg:
Done in 48 ms!
2.837 KiB - 27.4% = 2.06 KiB

delivery-failed.svg:
Done in 32 ms!
2.58 KiB - 13.6% = 2.23 KiB

invalidated.svg:
Done in 15 ms!
0.992 KiB - 50.9% = 0.487 KiB

lost.svg:
Done in 2 ms!
0.991 KiB - 50.8% = 0.487 KiB

order-placed.svg:
Done in 10 ms!
1.179 KiB - 25.7% = 0.876 KiB

order-cancelled.svg:
Done in 5 ms!
0.563 KiB - 34% = 0.371 KiB

insufficient-address.svg:
Done in 21 ms!
4.682 KiB - 49.6% = 2.359 KiB

payment-pending.svg:
Done in 3 ms!
0.912 KiB - 18.6% = 0.742 KiB

preparing-to-ship.svg:
Done in 8 ms!
1.178 KiB - 25.6% = 0.876 KiB

packing-order.svg:
Done in 6 ms!
1.364 KiB - 35.2% = 0.884 KiB

pickup-failed.svg:
Done in 9 ms!
1.452 KiB - 37% = 0.915 KiB

payment-expired.svg:
Done in 3 ms!
0.688 KiB - 29.2% = 0.487 KiB

payment-cancelled.svg:
Done in 4 ms!
0.981 KiB - 16.3% = 0.821 KiB

payment-failed.svg:
Done in 4 ms!
0.983 KiB - 20.4% = 0.783 KiB

return-booking-failed.svg:
Done in 3 ms!
0.991 KiB - 50.8% = 0.487 KiB

return-booked.svg:
Done in 2 ms!
0.669 KiB - 30.2% = 0.467 KiB

shipped.svg:
Done in 8 ms!
2.162 KiB - 27.6% = 1.564 KiB

return-requested.svg:
Done in 1 ms!
0.67 KiB - 30.3% = 0.467 KiB

with-driver.svg:
Done in 10 ms!
4.735 KiB - 54.3% = 2.163 KiB

returned-to-sender.svg:
Done in 2 ms!
0.669 KiB - 30.2% = 0.467 KiB

File: app/views/sephora_style_guide/home/_icons.slim [hunk 1]
```diff
@@ -97,68 +117,112 @@ ruby:
...
-      "name": "Order Placed",
-      "class": "order-placed"
-    },
+      "name": "With Customs",
+      "class": "customs"
+    }
     {
-      "name": "Packing Order",
-      "class": "packing-order"
+      "name": "Return Requested",
```
Comment (ycewe) [inline app/views/sephora_style_guide/home/_icons.slim]: R after W XD

File: app/views/sephora_style_guide/home/_icons.slim [hunk 2]
```diff
@@ -69,24 +69,44 @@ ruby:
...
     },
     {
-      "name": "Payment Paid",
-      "class": "payment-paid"
+      "name": "Customs Awaiting Payment",
+      "class": "customs_awaiting_payment"
+    },
+    {
+      "name": "Customs Failed",
+      "class": "customs_failed"
```
Comment (ycewe) [inline app/views/sephora_style_guide/home/_icons.slim]: kebab case?

File: app/views/sephora_style_guide/home/_icons.slim [hunk 3]
```diff
@@ -69,24 +69,44 @@ ruby:
...
-      "name": "Payment Cancelled",
-      "class": "payment-cancelled"
+      "name": "Completed",
+      "class": "completed"
     },
     {
-      "name": "Payment Paid",
-      "class": "payment-paid"
+      "name": "Customs Awaiting Payment",
+      "class": "customs_awaiting_payment"
```
Comment (ycewe) [inline app/views/sephora_style_guide/home/_icons.slim]: kebab case?

**Pull request merged**